app/src/main/res/menu/activity_generic_left_menu.xml

```diff
@@ -24,19 +24,25 @@
                 android:id="@+id/action_contact_us"
                 android:orderInCategory="400"
                 android:title="@string/menu_contact_us" />
+        </menu>
+    </item>
 
+    <item
+        android:orderInCategory="100"
```

Comment:
if we are actually utilizing `orderInCategory`, then it shouldn't have the same value as other items at the same level

Reply:
https://github.com/CruGlobal/godtools-android/pull/803/files#diff-c09f752f83501a05f88e2ed5ca34a244R4